design/ceph/ceph-jaeger-tracing.md

```diff
@@ -0,0 +1,30 @@
+---
+title: deploy jaeger tracing components to ceph cluster
+target-version: release-1.8
+---
+
+# Tracing - Jaeger
+
+## Summary
+
+In order to make it possible to trace in Ceph, we would need the components of jaeger tracing to be deployed in the ceph cluster.
+
+the jaeger components consists of:
+1. elasticsearch - needs to be manually deployed.
+2. jaeger collector, agent and UI - are all managed by the jaeger Operator.
+
+
+### Goals
+
+to be able to see traces from the ceph cluster (rgw and osd) in the jaeger UI.
+
+## Proposal details
+
+When Ceph is deployed via Rook, we would like to add a parameter to the rook CR that will indicate to enable tracing and deploy the jaeger resources.
```

Comment:
please provide a sample ceph cluster yaml with the new set of parameters you plan on adding

Reply:
please also provide an example for the generated yaml of the jaeger CR that our operator is expected to emit if tracing is configured.
showing the default values and setup that we plan on using